content/articles/creating-movies-and-animations-using-matlab/index.md

```diff
@@ -0,0 +1,243 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /creating-movies-and-animations-using-matlab/
+title: Creating movies and animations using Matlab
+description: In this article, we will look at how you can create an animation using Matlab. We will also look at the various steps involved and use the Matlab inbuilt functions to make the activity simpler.
+author: florence-atieno
+date: 2021-10-09T00:00:00-11:30
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/creating-movies-and-animations-using-matlab/hero.jpg
+    alt: Creating movies and animations using Matlab Hero image
+---
+
+Animation is a series of still images one after another. If you show these images together in rapid succession, the brain interprets them as continuous fluid motion.
+<!--more-->
+The animation follows a similar workflow to that of creating a flip-flop book. A flip-flop book is a booklet with a series of images that gradually change from one page to the next. 
+
+When you view the pages quickly, the images appear to animate by simulating motion or some other change. Animation has a wide advantage and is widely used in the science and engineering field. 
+
+It helps to bring ideas into real-life or give the context of the idea. In this article, we will look at how you can create an animation using Matlab. 
+
+We will also look at the various steps involved and use the Matlab inbuilt functions to make the activity simpler.
+
+### Prerequisites
+To follow along with this tutorial, you will need:
+- [MATLAB](https://www.mathworks.com/products/get-matlab.html?s_tid=gn_getml) installed.
+- Proper understanding of [MATLAB](https://www.section.io/engineering-education/getting-started-with-matlab/) basics.
+
+Animation in Matlab follows a workflow like that of creating a flipbook. The steps involved in creating an animation in Matlab are as follows:
+1. Run a simulation or generate data.
+2. Draw/render the scenario at some time `t_k`.
+3. Take a snapshot of the scenario.
+4. Advance time `t_k` to `t_(k+1)`.
+
+We will then, Go back and repeat steps 2 to 4 and keep going through building one frame or one page in the flipbook one at a time and saving it to the large flipbook. This means that the last step is:
```

Comment:
Go should be lower case. Remove the comma.

Would be better for flow to move the 5 next to the other steps and have the text underneath. Something like: Steps 2 and 4 will be repeated (rest of what you've said) then we will proceed to Step 5.